Doc/library/typing.rst

```diff
@@ -1230,6 +1232,123 @@ These are not used in annotations. They are building blocks for creating generic
     ``covariant=True`` or ``contravariant=True``.  See :pep:`484` for more
     details.  By default, type variables are invariant.
 
+.. class:: TypeVarTuple
+
+    Type variable tuple. A specialised form of :class:`Type variable <TypeVar>`
```

Comment:
```suggestion
    Type variable tuple. A specialized form of :class:`Type variable <TypeVar>`
```
The document is mostly in American English so let's be consistent (you also have "parameterization" instead of "parameterisation" in the next line).